test/unit/input_template_renderer_test.rb

```diff
@@ -57,13 +65,17 @@
         end
 
         context 'with invocation specified' do
-          let(:result) do
-            renderer.invocation = nil # TODO
-            renderer.render
+
+          before do
+            @input_value = FactoryGirl.create(:template_invocation_input_value,
+                                              :template_invocation =>  template_invocation,
+                                              :template_input      => template.template_inputs.first,
```

Comment:
please don't indent hashes, this is not puppet \o/ (both lines above have extra spaces)